### ISSUE TITLE
Missing query strings after a live_redirect/2

### ISSUE THREAD

**Issue by hurty** [phoenixframework/phoenix_live_view#594]
### Environment

* Elixir version (elixir -v): 1.9.1
* Phoenix version (mix deps): 1.4.12
* Phoenix LiveView version (mix deps): ref df820fe
* NodeJS version (node -v): 12.12.0
* NPM version (npm -v): 6.13.1
* Operating system: MacOS 10.15.2

### Actual behavior
Since df820fe the query strings parameters are no longer pushed in the browser URL after a `live_redirect/2`. (They are however present on the liveview's side in `handle_params/3`)

### Expected behavior
Query strings should be visible in URL.

**Comment by josevalim** [comment]
@hurty can you please provide a sample app that reproduces the error? Or change the example app provided by @chrismccord. Thank you.

**Comment by hurty** [comment]
@josevalim Sure! You can take a look here: https://github.com/hurty/livebug

**Comment by chrismccord** [comment]
Please ensure you have blown away node_modules and re-installed as I am not able to recreate the issue with your sample app. Thanks!

**Comment by hurty** [comment]
@chrismccord I've deleted node_modules and re-installed but I still got the problem. Weird... 🤔 

Just to make sure we are looking at the same thing: after a live_redirect (now push_patch) instead of having `http://localhost:4000/?query=hello` in the browser URL bar, I simply got `http://localhost:4000` with no query string.
The problem is just in the URL shown, the rest seems to work fine.

I've updated the sample app with the latest dependencies.
